oio/account/server.py

```diff
@@ -365,10 +365,11 @@ def on_account_containers(self, req):
         prefix = req.args.get('prefix', '')
         limit = int(req.args.get('limit', '1000'))
         delimiter = req.args.get('delimiter', '')
+        buckets = bool(req.args.get('buckets', False))
```

Comment:
We should use `true_value` or `boolean_value` (https://github.com/open-io/oio-sds/blob/master/oio/common/easy_value.py#L44)